PyOpenWorm/context.py

```diff
@@ -57,7 +57,8 @@ def contextualize_class_augment(self, context):
         if context is None:
             return self
         ctxd_meta = contextualize_metaclass(context, self)
-        res = ctxd_meta(self.__name__, (self,), dict(class_context=context.identifier))
+        res = ctxd_meta(self.__name__, (self,), dict(
```

Comment:
Please see: https://github.com/openworm/PyOpenWorm/blob/dev/docs/coding_standards.rst

No need to wrap lines less than 120 characters or to apply pep8 where you aren't making a functional change.